pages/cart-page/cart-page.css

```diff
@@ -0,0 +1,72 @@
+@import url('/main.css');
+@import url('https://fonts.googleapis.com/css2?family=Prompt:ital@1&display=swap');
+
+.grid-wrapper{
+    display: grid;
+    grid-template-columns: 6fr 4fr;
+    grid-template-rows: auto auto 1fr auto;
+}
+.navbar{
+    grid-column: 1/3;
+    grid-row: 1/2;
```

Comment:
same here once check

Reply:
Same comment as above. I need to reconsider, your thoughts?